### PR TITLE
Removes drupal function from auth validation

### PR DESCRIPTION
**Pull request by deadlybutter**
#### What's this PR do?
Fixes the validations to show the correct messages! Also updated the JS syntax

#### How should this be reviewed?
When you go to the create an account form do you see the proper validation messages?

### CODE REVIEW

File: resources/assets/js/validators/auth.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
 define(function(require) {
-  "use strict";
+  'use strict';
```
Comment (deadlybutter) [inline resources/assets/js/validators/auth.js]: this one needs to be double quotes, got caught in my replace all

Comment (DFurnes) [inline resources/assets/js/validators/auth.js]: How come?

Comment (DFurnes) [inline resources/assets/js/validators/auth.js]: I think Babel actually automatically inserts the strict-mode directive when compiling the code, so you could probably remove it altogether (and get rid of the old AMD-style `define` wrapper).

Comment (deadlybutter) [inline resources/assets/js/validators/auth.js]: Sure thing

Comment (deadlybutter) [inline resources/assets/js/validators/auth.js]: And functionality i don't think it makes a difference, I just always saw quotes as the "convention"

**Comment by jessleenyc** [comment]
[(5) As a user, I want to see validation messaging on the Birthday field that makes sense so that I don't see "Cool @age!"](https://trello.com/c/xZPAqIz8/242-5-as-a-user-i-want-to-see-validation-messaging-on-the-birthday-field-that-makes-sense-so-that-i-don-t-see-cool-age)

**Pull request merged**